packages/p2p/src/components/my-ads/my-ads.jsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './my-ads.scss';
-import { localize } from 'deriv-translations';
+// import { localize } from 'deriv-translations';
+import ToggleAds from './toggle-ads.jsx';
+
+class MyAds extends Component {
+    constructor () {
+        super();
+        this.state = {
+            is_enabled: false,
+        };
+        this.setEnabled = this.setEnabled.bind(this);
+    }
+
+    setEnabled (is_enabled) {
+        this.setState(
+            {
+                is_enabled,
+            }
+        );
+    }
```

Comment:
the constructor and setEnabled function should be in the toggleAds, that was fine. the initial is_enabled flag should be retrieved from the API and passed to toggleAds here, that's what I meant, let's discuss it if there's any confusion